tools/cjxl_main.cc

```diff
@@ -322,6 +322,12 @@ struct CompressArgs {
 
     cmdline->AddHelpText("\nOptions for experimentation / benchmarking:", 3);
 
+    cmdline->AddOptionValue('\0', "noise", "0|1",
+                            "Force disable/enable adaptive noise generation "
+                            "(experimental). Default "
+                            "is 'encoder chooses'",
```

Comment:
Isn’t the default “no”?

Reply:
Yes, the current encoder choice is always "no" :).

In the API the default is also described as "encoder chooses". At some point the encoder choice might become something a bit more interesting than just "always choose no".

Reply:
Eh, fair, I suppose 😁